src/main/cms/cms_menu_power.c

```diff
@@ -100,19 +100,19 @@ static const OSD_Entry cmsx_menuPowerEntries[] =
 {
     { "-- POWER --", OME_Label, NULL, NULL, 0},
 
-    { "V METER", OME_TAB, NULL, &(OSD_TAB_t){ &batteryConfig_voltageMeterSource, VOLTAGE_METER_COUNT - 1, voltageMeterSourceNames }, 0 },
-    { "I METER", OME_TAB, NULL, &(OSD_TAB_t){ &batteryConfig_currentMeterSource, CURRENT_METER_COUNT - 1, currentMeterSourceNames }, 0 },
+    { "V METER", OME_TAB, NULL, &(OSD_TAB_t){ &batteryConfig_voltageMeterSource, VOLTAGE_METER_COUNT - 1, voltageMeterSourceNames }, REBOOT_REQUIRED },
+    { "I METER", OME_TAB, NULL, &(OSD_TAB_t){ &batteryConfig_currentMeterSource, CURRENT_METER_COUNT - 1, currentMeterSourceNames }, REBOOT_REQUIRED },
```

Comment:
Do all of these really need a reboot?

Reply:
No sorry, I mainly thought that they affected to real time current/voltage reads, but that is not the case.

Reply:
Can you please point out what reference to `batteryConfig()->voltageMeterSource` and `batteryConfig()->vbatmaxcellvoltage` means that a reboot is required for changes to them to take effect?

The reality is that we are trying to move away from _any_ settings changes requiring a reboot - most of them have been refactored to not require a reboot, and we should try to not muddy the waters by adding required reboots for settings that don't need them.

Reply:
@mikeller Hi i noticed this update in 4.1.5 betaflight Target: TMOTORF7 
Manufacturer: TMTR 
Version: 4.1.5 
Binary: betaflight_4.1.5_STM32F7X2.hex 
Date: 16-03-2020 16:50 

I am not sure if this is related, but i posted an issue .. WHat happens is Battery readings freeze as soon as quad is armed, readings dont happen i can only see the original battery level. If i unplug battery and re-plug i will only then see the new updated batt levels, again if i re-arm the quad voltage meters freeze again. please help me as i  am close to damaging lipos. I use the T-Motor HD F7 Stack and 55a 4in1 32bit esc + DJI system